core/services/ocr2/plugins/ocr2keeper/evm21/streams_lookup.go

```diff
@@ -593,3 +595,31 @@ func (r *EvmRegistry) generateHMAC(method string, path string, body []byte, clie
 	userHmac := hex.EncodeToString(signedMessage.Sum(nil))
 	return userHmac
 }
+
+// calculateRetryConfig returns retryable and plugin retry interval based on how many times plugin has retried this work
+func (r *EvmRegistry) calculateRetryConfig(retryable bool, prk string) (bool, time.Duration) {
+	var ri time.Duration
+	if retryable {
+		var retries int
+		totalAttempts, ok := r.mercury.pluginRetryCache.Get(prk)
+		if ok {
+			retries = totalAttempts.(int)
```

Comment:
is there a way we can make cache strongly typed?

Reply:
most cache tries to accept an interface to keep its flexibility so it's not possible